test/docfx.E2E.Tests/DocfxSeedSiteTest.cs

```diff
@@ -281,8 +281,7 @@ public void TestXRefMap()
         private void TestPageCommon()
         {
             // check logo
-            IWebElement element = _driver.FindElement(By.Id("logo"));
-            Assert.Equal("svg", element.TagName);
+            Assert.NotEmpty(_driver.FindElements(By.Id("logo")));
```

Comment:
In page source, it's a `img` tag with id = "logo". When finishing loading, it will become a `svg` tag. There's a chance to fail when tagName is still img.